storybook/manager.tsx

```diff
@@ -3,7 +3,7 @@
 import React from "react"
 import { addons } from "@storybook/addons"
 import { API_SidebarOptions as SidebarOptions } from "@storybook/types"
-import { defaultTheme } from "@kaizen/design-tokens"
+import { tokens } from "../packages/design-tokens/src/js"
```

Comment:
Not sure why `@kaizen/design-tokens/js` doesn't work...

Reply:
Would `~design-tokens/js` work? Seems we had made an alias

Reply:
Also doesn't work, I suspect we have a workspaces issue...

Reply:
Works as a package again